src/components/app/Home.js

```diff
@@ -78,6 +102,44 @@ class UploadButton extends React.PureComponent<UploadButtonProps> {
           />
           Select a profile to open
         </label>
+        <label
+          id="LoadFromUrlButton"
+          className="homeSectionButton"
+          onClick={this._loadFromUrlPressed}
+        >
+          Load profile from an URL
+        </label>
+        {this.state.isLoadFromUrlPressed ? (
+          <UploadFromUrl {...this.props} />
+        ) : null}
+      </div>
+    );
+  }
+}
+
+class UploadFromUrl extends React.PureComponent<UploadFromUrlProps> {
+  _input: HTMLInputElement | null;
+
+  _takeInputRef = input => {
+    this._input = input;
+  };
+
+  _upload = () => {
+    if (this._input) {
+      this.props.triggerLoadingFromUrl(this._input.value);
+    }
+  };
+  render() {
+    return (
+      <div className="homeSectionUploadFromUrl">
+        <input
+          className="homeSectionUploadFromUrlInput"
+          type="url"
+          ref={this._takeInputRef}
+        />
+        <label className="homeSectionButton" onClick={this._upload}>
```

Comment:
Instead of using `onClick`, please use a `onSubmit` on the `form` above, so that a user can start the upload by just pressing `enter` (and submitting the form then)